ocaml/xapi/certificates.ml

```diff
@@ -80,6 +80,9 @@ let pp_hash hash =
   in
   String.init length value_of
 
+let pp_fingerprint_of_cert ~hash_type cert =
+  X509.Certificate.fingerprint hash_type cert |> pp_hash
+
```

Comment:
This works as well, and is shorter:

```suggestion
let pp_fingerprint ~hash cert =
  X509.Certificate.fingerprint hash cert |> pp_hash
```

Reply:
Good point. Should it still be ~hash_type though, as it's specifying sha256/sha1, it isn't a hash itself?